docs(stack): reorganize navigation into evaluate and deployment paths (#1605)
Consensys/doc.linea: docs/stack/how-it-works/trust-model.mdx

---
title: Trust and responsibilities
description: >-
  Trust assumptions, control responsibilities, and failure modes for Lineth
  deployments
sidebar_position: 3
image: /img/socialCards/trust-model.jpg
---

Use this page to evaluate who controls what in a Lineth (formerly the Linea Stack) deployment, what participants can verify, and what happens when dependencies fail.

> What this page covers: who operates each component, what each component can do unilaterally, what participants can independently verify, what happens when something fails, and how trust assumptions change across deployment models.

Reviewer: ```suggestion
This page describes who operates each Lineth component, what each component can do, what participants can verify, what happens when dependencies fail, and how trust assumptions change across deployment models.
```

> What this page does not cover: configuration flags, deployment topology, or component internals. For those, see [Deployment components](./core-components) and [Deployment models](./deployment-models).

Reviewer: ```suggestion
For more information on deployment topology, see [Deployment components](./core-components) and [Deployment models](./deployment-models).
```

I don't think we need to go into detail on what each page doesn't cover.
